jsonnet/kube-prometheus/kube-prometheus-insecure-kubelet.libsonnet

```diff
@@ -9,6 +9,12 @@
               scheme: 'http',
               interval: '30s',
               bearerTokenFile: '/var/run/secrets/kubernetes.io/serviceaccount/token',
+              relabelings: [
```

Comment:
I'm not sure where this file is used, but I figured I'd update it for consistency's sake.